front/package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
delete this file, it is not relevant in the front

Reply:
Okay, done